src/components/transaction/TransactionTable/index.tsx

```diff
@@ -0,0 +1,294 @@
+      flex: 1;
+      width: 100%;
+      justify-content: space-between;
+      margin: 0;
+      margin-bottom: 18px;
+      min-height: 32px;
+      span.span-inside-tooltip {
+        align-items: flex-end;
+        flex-direction: column;
+        img {
+          margin-left: 0;
+        }
+      }
+    }
+    .header-value {
+      flex-wrap: wrap;
+      text-align: end;
+    }
+    .span-copybtn-wrap {
+      display: flex;
+      flex-wrap: nowrap;
+      span {
+        display: flex;
+        align-items: center;
+      }
+      .copy-text {
+        margin-left: 5px;
+      }
+    }
+  }
+  overflow: auto;
+`
+
+const HeaderTitle = styled.span`
+  display: none;
+  ${media.desktopMediumDown} {
+    font-weight: 600;
+    align-items: center;
+    display: flex;
+    margin-right: 3rem;
+    svg {
+      margin-left: 5px;
+    }
+  }
+`
+const HeaderValue = styled.span`
+  ${media.desktopMediumDown} {
+    flex-wrap: wrap;
+    text-align: end;
+  }
+`
+
+function getLimitPrice(order: Order, isPriceInverted: boolean): string {
+  if (!order.buyToken || !order.sellToken) return '-'
+
+  const calculatedPrice = getOrderLimitPrice({
+    buyAmount: order.buyAmount,
+    sellAmount: order.sellAmount,
+    buyTokenDecimals: order.buyToken.decimals,
+    sellTokenDecimals: order.sellToken.decimals,
+    inverted: isPriceInverted,
+  })
+
+  return formatCalculatedPriceToDisplay(calculatedPrice, order.buyToken, order.sellToken, isPriceInverted)
+}
+
+const tooltip = {
+  orderID: 'A unique identifier ID for this order.',
+}
+
+export type Props = StyledUserDetailsTableProps & {
+  orders: Order[]
+}
+
+interface RowProps {
+  order: Order
+  isPriceInverted: boolean
+  invertLimitPrice: () => void
+}
+
+const RowTransaction: React.FC<RowProps> = ({ order, isPriceInverted, invertLimitPrice }) => {
+  const {
+    buyToken,
+    buyAmount,
+    expirationDate,
+    partiallyFilled = false,
+    sellToken,
+    sellAmount,
+    kind,
+    txHash,
+    shortId,
+    uid,
+  } = order
+  const network = useNetworkId()
+  const buyTokenSymbol = buyToken ? safeTokenName(buyToken) : ''
+  const sellTokenSymbol = sellToken ? safeTokenName(sellToken) : ''
+  const sellFormattedAmount = formattedAmount(sellToken, sellAmount)
+  const buyFormattedAmount = formattedAmount(buyToken, buyAmount)
+  const renderSpinnerWhenNoValue = (textValue: string): JSX.Element | void => {
+    if (textValue === '-') return <FontAwesomeIcon icon={faSpinner} spin size="1x" />
+  }
+  const limitPriceSettled = getLimitPrice(order, isPriceInverted)
+
+  return (
+    <tr key={txHash}>
+      <td>
+        <HeaderTitle>
+          Order ID <HelpTooltip tooltip={tooltip.orderID} />
+        </HeaderTitle>
+        <HeaderValue>
+          <RowWithCopyButton
+            className="span-copybtn-wrap"
+            textToCopy={uid}
+            contentsToDisplay={
+              <LinkWithPrefixNetwork to={`/address/${order.txHash}`} rel="noopener noreferrer" target="_blank">
```

Comment:
Not quite the right link yet.

The correct should be:

```suggestion
              <LinkWithPrefixNetwork to={`/orders/${order.uid}`} rel="noopener noreferrer" target="_blank">
```